package.json

```diff
@@ -78,10 +80,10 @@
     "@babel/node": "^7.2.2",
     "@babel/preset-env": "^7.3.4",
     "@testing-library/dom": "^7.11.0",
-    "cypress": "^4.5.0",
+    "cypress": "^6.2.1",
```

Comment:
This will break some of the things we run in QE, since we share the same package json. Still this will just impact the phase1 job

Reply:
It's on the TODO "update existing code to the new Cypress". I believe, we will need to especially move from the use of `route()` to interceptors there.